### PR TITLE
RF_00.002.04 | Create Pipeline Project with an empty item name field

### PR DESCRIPTION
**Pull request by Natalibu1**
RF_00.002.04 | New Item > Create Pipeline Project | Create Pipeline Project with an empty item name field
[#455](https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/455)
Converted TC_00.002.04 to POM

### CODE REVIEW

File: cypress/e2e/newItemCreatePipelineProjectPOM.cy.js [hunk 1]
```diff
@@ -28,11 +28,20 @@ describe("US_00.002 | New Item > Create Pipeline Project", () => {
...
-        .typeNewItemName(randomItemName)
-        .selectPipelineProject()
+      .clickNewItemMenuLink()
+      .typeNewItemName(randomItemName)
+      .selectPipelineProject()
 
     newJobPage.getPipelineSelectedState()
-        .should('have.attr', 'aria-checked', 'true')
+      .should('have.attr', 'aria-checked', 'true')
   });
```
Comment (Negrescu29) [inline cypress/e2e/newItemCreatePipelineProjectPOM.cy.js:37]: @Natalibu1 not sure why the indentation changed, do you have a code formatter?

File: cypress/e2e/newItemCreatePipelineProjectPOM.cy.js [hunk 2]
```diff
@@ -28,11 +28,20 @@ describe("US_00.002 | New Item > Create Pipeline Project", () => {
...
+
+  it('TC_00.002.04 | Create Pipeline Project with an empty item name field', () => {
+
+    dashboardPage.clickNewItemMenuLink();
+    newJobPage.selectPipelineProject()
```
Comment (Negrescu29) [inline cypress/e2e/newItemCreatePipelineProjectPOM.cy.js:42]: lgtm. 

just as a best practice add a space between steps and assertation.

**Review by Negrescu29 — APPROVED** [review]

File: cypress/e2e/newItemCreatePipelineProjectPOM.cy.js (hunk 1 above)
Comment (Natalibu1) [reply cypress/e2e/newItemCreatePipelineProjectPOM.cy.js:37]: It's just a standard Shift+Alt+F

File: cypress/e2e/newItemCreatePipelineProjectPOM.cy.js [hunk 3]
```diff
@@ -5,12 +5,12 @@ import DashboardPage from "../pageObjects/DashboardPage";
...
 
 import allKeys from "../fixtures/newJobPageData.json";
-import {newItem} from "../fixtures/messages.json";
+import { newItem } from "../fixtures/messages.json";
 
 const dashboardPage = new DashboardPage();
 const newJobPage = new NewJobPage();
 
-const {projectNameInvalid, errorMessageColor} = allKeys;
+const { projectNameInvalid, errorMessageColor } = allKeys;
```
Comment (adiuzhikov) [inline cypress/e2e/newItemCreatePipelineProjectPOM.cy.js:13]: Please, stop formatting the whole document. You have Format Selection option.

**Comment by Natalibu1** [comment]
> The test passed and it seems to be ok. Concerning your github board card: could you please bring it in order and fill some necessary fields:
> 
> * Assignees;
> * Card Type;
> * PR link.

Done

**Comment by ivanlashkevich** [comment]
> > The test passed and it seems to be ok. Concerning your github board card: could you please bring it in order and fill some necessary fields:
> > 
> > * Assignees;
> > * Card Type;
> > * PR link.
> 
> Done

Good job! Approved.

**Review by ivanlashkevich — APPROVED** [review]

**Pull request merged**